src/main/java/uk/gov/hmcts/reform/draftstore/DraftStoreApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
 
 @SpringBootApplication
+@EnableScheduling
```

Comment:
Looking at the current setup it appears that this scheduled job would start on all nodes?
since its deleting it shouldn't be a problem? but worth calling out

Reply:
Yeah, no danger of doing something more than once in this case :)